numpyro/distributions/continuous.py

```diff
@@ -1033,6 +1033,33 @@ def icdf(self, q):
         return self.loc + self.scale * logit(q)
 
 
+class LogUniform(TransformedDistribution):
```

Comment:
You can see docs of some distributions [here](https://num.pyro.ai/en/stable/distributions.html#numpyro.distributions.continuous.LKJ).